core/UrlHelper.php

```diff
@@ -155,6 +155,14 @@ public static function getArrayFromQueryString($urlQuery)
         if (strlen($urlQuery) == 0) {
             return array();
         }
+
```

Comment:
I had it in caller level first, but this way it is simply cached everywhere as it is called by different methods eg during queued tracking 

59,937 times by `Piwik\UrlHelper::getParameterFromQueryString`
39,353 times by `Piwik\Plugins\Actions\Actions\ActionSiteSearch::detectSiteSearchFromUrl`
9,974 times by `Piwik\Tracker\PageUrl::excludeQueryParametersFromUrl`

Ideally, it would be not in here but I think there shouldn't be many downsides in this case.